v2/pages/adminAssessment/adminAssessment.sql

```diff
@@ -23,6 +23,44 @@ WHERE
     AND q.deleted_at IS NULL
 ORDER BY z.number, z.id, aq.number;
 
+-- BLOCK question_stats
+SELECT
+    sub.question_id,
+    avg(sub.mean_score_per_user_per_question) as mean_score_per_question,
+    sub.number,
+    corr(sub.mean_score_per_user_per_question, sub2.average_assessment_score_for_user) as discrimination
+FROM (
+    SELECT
+        assessment_instances.user_id,
+        assessment_questions.question_id,
+        assessment_questions.number,
+        avg(instance_questions.points/assessment_questions.max_points) as mean_score_per_user_per_question
```

Comment:
I know you said that we incorrectly have zero for `score_perc` in some cases. I would rather fix that and use `score_perc` here rather than `points/max_points`. I'll send you a recent DB dump and you can check whether in recent assessments we have `score_perc` correctly calculated.